cdap-operational-stats-core/src/main/java/co/cask/cdap/operations/hdfs/HDFSInfo.java

```diff
@@ -86,8 +90,13 @@ public String getWebURL() {
           }
         }
       }
-    } catch (IOException e) {
-      LOG.warn("Error in determining HDFS URL. Web URL of HDFS will not be available in HDFS operational stats.", e);
+      lastCollectFailed = false;
+    } catch (Exception e) {
+      // TODO: remove once CDAP-7887 is fixed
```

Comment:
isn't CDAP-7887 fixed as a part of this? So this can be removed?
Or is this comment incorrect/misleading?